modules/mobile/app/models/mobile/v0/service.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require 'common/models/resource'
+
+module Mobile
+  module V0
+    class Service < Common::Resource
```

Comment:
I think we're back in one of those positions like using the word "request" in appointment request. Service will be a confusing word, but it looks like that's what's used in the upstream service and in other parts of the rails app. I've seen in called "specialty" in another upstream service. "Medical service" would be better than service, IMO.

Reply:
That's a good point. Service could mean something drastically different in this code base depending on where it's used so I'll change it to medical service.